### DISCUSSION TITLE
Please, is possible to add an option for randomize presets selection?

### DISCUSSION THREAD

**Discussion by DavideAlidosi** [AIrjen/OneButtonPrompt#166] [Ideas]
In my workflow I make extensive use of the Gernerate Forever option, so I would find it very useful to add an option to randomize the selection of Presets.
This would allow me to let the workstation work in batch and obtain a large variety of images.
Thank you and congratulations on the great script you are writing.

**Comment by AIrjen** [comment]
Hi, that is an excellent suggestion! Will do this somewhere in the coming days.

**Comment by AIrjen** [comment]
OK done! You can get it in the latest version, both in A1111 and ComfyUI versions (will add to RuinedFooocus shortly)

![image](https://github.com/AIrjen/OneButtonPrompt/assets/130234949/7d0d4d07-478c-4a2d-88ae-009877c6165e)

Enjoy!

**Reply by DavideAlidosi** [reply]
Thank's man, this is super!!!!!!! I really appreciate it, if there is anything I can do please tell me.
